R/mergingandstackingutilities.R

```diff
@@ -76,6 +76,7 @@ createReadErrorHandler <- function(data.set.name)
 #' @importFrom flipAPI QSaveData IsDisplayrCloudDriveAvailable
 writeDataSet <- function(data.set, data.set.name, is.saved.to.cloud)
 {
+    data.set <- sanitizeSPSSVariableNames(data.set)
```

Comment:
This needs to be done outside `writeDataSet` as the modified names need to be used elsewhere.

Reply:
It might be best to place sanitizeSPSSVariableNames inside https://github.com/Displayr/flipData/blob/master/R/mergedatasetsbycase.R#L1760
https://github.com/Displayr/flipData/blob/master/R/mergedatasetsbyvariable.R#L372
https://github.com/Displayr/flipData/blob/master/R/stacking.R#L1051